receiver/hostmetricsreceiver/README.md

```diff
@@ -141,6 +141,14 @@ service:
       receivers: [hostmetrics, hostmetrics/disk]
 ```
 
+## Resource attributes
+
+Currently, the hostmetrics receiver does not set any Resource attributes on the exported metrics. However, if you want to set Resource attributes, you can provide them via environment variables via the [resourcedetection](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor#environment-variable) processor. For example, you can add the following resource attributes to adhere to [Resource Semantic Conventions](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/):
+
+```
+export OTEL_RESOURCE_ATTRIBUTES="service.name=website,service.namespace=dev,service.instance.id=627cc493-f310-47de-96bd-71410b7dec09"
```

Comment:
Can you replace the values with representative values like `service.name=<the name of your service>`.  I don't want anyone to read this and think these are the semantic conventions for resource attributes.